### ISSUE TITLE
Settings screen missing heading

### ISSUE THREAD

**Issue by lutrov** [themegrill/easy-social-sharing#5]
Just a gentle reminder that Wordpress has a standard for admin screen headings. You don't seem to be using one on your settings screen.

https://make.wordpress.org/core/2015/10/28/headings-hierarchy-changes-in-the-admin-screens/

**Comment by shivapoudel** [comment]
@lutrov Its been there but with a class `screen-reader-text`. See: https://github.com/themegrill/easy-social-sharing/blob/master/includes/admin/views/html-admin-settings.php#L33

**Comment by lutrov** [comment]
Yeah but I don't think that's the right place for it, is it? Shouldn't it just above the <form> element? It it a page header, after all, and right now, the settings screen shows no header which doesn't look right. Wordpress admin screens should always have a visible page header.

**Comment by shivapoudel** [comment]
If we append it just after `<div class="wrap easy-social-sharing">` then ESS message and other plugins notice is moving above h1 in our setting pages like this: 
![capture](https://user-images.githubusercontent.com/3774827/28764456-17a02cd2-75e5-11e7-9fce-1b8be88802f3.PNG)
Propose a pull request if you have the right fix :)

**Comment by lutrov** [comment]
From that screenshot, I agree that the screen notice should be appearing below the main page heading, which says "General Options" in this case. But the main heading should also be the first thing on the content area, not below the tabs. So the order should be:

1. The main heading
2. The update, error, info, etc notice
3. The tabs

I don't have the right fix because I don't have the time to look at all your code, let alone modify & test it. I was hoping to just alert you to the fact your plugin needs modification to make it WP 4.4+ compatible.
